en/SUMMARY.md

```diff
@@ -313,7 +313,6 @@
       * [Ubuntu Setup](dev_setup/dev_env_linux_ubuntu.md)
       * [Windows Setup](dev_setup/dev_env_windows_wsl.md)
       * [Visual Studio Code IDE](dev_setup/vscode.md)
-      * [Fast DDS installation](dev_setup/fast-dds-installation.md)
```

Comment:
Note, this was how to install the old toolchain. We will need to replaces this with a new doc. I have deleted the doc. I have not redirected - just created one redirection for the main document.